### PR TITLE
Alonzo

### PR DESCRIPTION
**Pull request by alonzoalcantar**
Added crud functionality to user models, refactored the profile page, and added icons to search and menu tabs

### CODE REVIEW

File: main_app/templates/coach/detail.html [hunk 1]
```diff
@@ -15,10 +15,12 @@ <h1>{{ coach.name }}</h1>
         <p>Specialty: {{ coach.specialty }}</p>
         <p>Schedule/Availability: {{ coach.availability }}</p>
         </div>
+        {% if perms.main_app.add_business %}
```
Comment (Joseph-King93) [inline main_app/templates/coach/detail.html:18]: I'm fine with leaving perm lock as "add_business" even though this is for coach instead.

Functionally, it does the same thing. Only an upgraded account can do XYZ, including any and all business crud.

Leave as is?

**Review by Joseph-King93 — CHANGES_REQUESTED** [review]
Please let me know what you think about in-line comment before approval.

File: main_app/templates/coach/detail.html (hunk 1 above)
Comment (alonzoalcantar) [reply main_app/templates/coach/detail.html:18]: yeah i realized that as-well when i was trying to test the app. I wasn't sure how to add the 'coach' permissions when the account is upgraded so i worked with what you already had set up. after adding the perm lock the functionality seemed to be the same so i think its fine to just leave it as it .

Comment (Joseph-King93) [reply main_app/templates/coach/detail.html:18]: can be changed later if need be. Will approve now as functionality is working as intended! Thanks!

**Review by Joseph-King93 — APPROVED** [review]

**Pull request merged**